code/go/pkg/validator/validator_test.go

```diff
@@ -90,6 +90,7 @@ func TestValidateFile(t *testing.T) {
 				"field services.docker-custom-agent: Must not validate the schema (not)",
 			},
 		},
+		"visualizations_by_reference": {},
```

Comment:
Adding this new element here, the package will be tested, but no errors can be checked. Should it be moved to another test ?

Reply:
Maybe we can have some undocumented environment variable like `PACKAGE_SPEC_WARNINGS_AS_ERRORS` that produce errors on warnings. We could enable it in tests.

Reply:
Added that environment variable with some helpers to be able to do tests on warnings
https://github.com/mrodm/package-spec/commit/06f27a9a8f7bc212ad4017cbbae45b8d43f6a4e8